Qt/mainwindow.cpp

```diff
@@ -598,10 +608,13 @@ void MainWindow::createMenus()
 
 	// Help
 	MenuTree* helpMenu = new MenuTree(this, menuBar(),    QT_TR_NOOP("&Help"));
-	helpMenu->add(new MenuAction(this, SLOT(websiteAct()),    QT_TR_NOOP("Official &website"), QKeySequence::HelpContents));
-	helpMenu->add(new MenuAction(this, SLOT(forumAct()),      QT_TR_NOOP("Official &forum")));
-	helpMenu->add(new MenuAction(this, SLOT(gitAct()),        QT_TR_NOOP("&GitHub")));
-	helpMenu->add(new MenuAction(this, SLOT(aboutAct()),      QT_TR_NOOP("&About PPSSPP..."), QKeySequence::WhatsThis));
+	helpMenu->add(new MenuAction(this, SLOT(websiteAct()),    QT_TR_NOOP("Visit www.&ppsspp.org")));
+	helpMenu->add(new MenuAction(this, SLOT(forumAct()),      QT_TR_NOOP("PPSSPP &forums")));
+	helpMenu->add(new MenuAction(this, SLOT(goldAct()),       QT_TR_NOOP("Buy &Gold")));
```

Comment:
I have no strong opinion either way, but I don't think you can actually buy a "gold" version of PPSSPP Qt.  That said, it's just a compile flag so you could compile it...

Not sure it matters though, since it's just a donation essentially anyway.

-[Unknown]

Reply:
Why not? If somebody wants to donate, he may buy vesion for other platform. There is no reason not to show it and no other way to do it (like Patreon or smth).

Reply:
Sounds fair.

-[Unknown]